app/classes.py

```diff
@@ -230,7 +230,7 @@ def frac_required_intakes(self):
 class Intake(db.Model):
     id = db.Column(db.Integer, primary_key=True)
     s3_url = db.Column(db.String(500), unique=True, nullable=True)
-    recording_data = db.Column(db.JSON(), unique=False, nullable=True)
+    recording_data = db.Column(db.Text(), unique=False, nullable=True)
```

Comment:
Hmm...we'll have to change the schema on RDS as well.  At this stage it doesn't hurt to just blow away the DB and creating a new one but (note to self) I need to find a more sustainable solution for dealing with migrations / schema changes.